dev-libs/libical/libical-2.0.0.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An implementation of iCalendar protocols and data formats"
+DESCRIPTION="Libical is an Open Source implementation of the iCalendar \
+protocols and protocol data units. The iCalendar specification describes how \
+calendar clients can communicate with calendar servers so users can store \
+their calendar data and arrange meetings with other users."
+HOMEPAGE="http://libical.github.io/libical/"
+COPYRIGHT="Eric Busboom"
+LICENSE="MPL v1.0
+	GNU LGPL v2.1"
+REVISION="1"
+scrGitRev="6c4af23b0a95fd105f38e879908cbc80390f3716"
```

Comment:
@raefaldhia, the line @pulkomandy is implicitly requesting the removal is the one which defines srcGitRev, of course. -Sorry if this was obvious.)